src/odemis/acq/stream/_live.py

```diff
@@ -296,13 +300,17 @@ def _applyROI(self):
         Note: should only be called when active (because it directly modifies
           the hardware settings)
         """
-        res, trans = self._computeROISettings(self.roi.value)
+        need_res = True if hasVA(self._emitter, "resolution") and not self._emitter.resolution.readonly else False
+        need_trans = True if hasVA(self._emitter, "translation") and not self._emitter.translation.readonly else False
```

Comment:
The `v = True if condition else False` is a bit verbose, to the point where I'm finding it a little bit confusing because I'd expect such a complex writing is a sign of a special trick, but there is no such trick.
Typically, you'd just write `need_res = hasVA(self._emitter, "resolution") and not self._emitter.resolution.readonly`.
If you find it confusing, you could also try `need_res = bool(hasVA(self._emitter, "resolution") and not self._emitter.resolution.readonly)`, what do you think?

Reply:
I linke that implementation actually. I didn't think of it.
I have chosen for this one:
`need_res = hasVA(self._emitter, "resolution") and not self._emitter.resolution.readonly  # Boolean`